src/seedu/addressbook/AddressBook.java

```diff
@@ -578,6 +584,16 @@ private static String executeListAllPersonsInAddressBook() {
         showToUser(toBeDisplayed);
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
+    
+    /**
+     * Displays a list of persons in the address book alphabetically sorted by name
+     */
+    private static String executeSortPersons() {
+    	ArrayList<String[]> toBeSorted = getAllPersonsInAddressBook();
```

Comment:
Variable name probably can be better, as it may get confusing later in the code (i.e. after the content is sorted and we still call it toBeSorted may be misleading).